tools/src/debug/bin/list-process.ts

```diff
@@ -0,0 +1,48 @@
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import { PID, listProcess } from '../utils';
+import { ListProcessesBinOptions } from './types';
+
+const { commandMatch, pid, verbose, slice } = yargs(hideBin(process.argv))
+  .version(false)
+  .options({
+    verbose: { type: 'boolean' },
+    commandMatch: { type: 'string', array: true, default: [] },
+    pid: { type: 'string', array: true, default: [] },
+    slice: { type: 'number', default: 9 },
+  })
+  .coerce('commandMatch', (commandMatch: string[]) =>
+    commandMatch.flatMap(p => p.split(',')).filter(p => p !== ''),
+  )
+  .coerce('pid', (pid: string[]) =>
+    pid.flatMap(p => p.split(',')).filter(p => p !== ''),
+  ).argv as Omit<ListProcessesBinOptions, 'pid' | 'commandMatch' | 'slice'> & {
+  pid: PID[];
+  commandMatch: string[];
+} & {
+  slice: number;
+};
+
+verbose &&
+  commandMatch &&
+  console.log(`Command Match: ${commandMatch.join(', ')}`);
+verbose &&
+  pid &&
+  pid.length < 0 &&
+  console.log(`PID Filter: ${pid.join(', ')}`);
```

Comment:
Same as previous comment - please use `if`-statements here.

Also, how can `pid.length < 0` ever be true? I guess you meant `pid.length > 0`?